docs/source/documentation.rst

```diff
@@ -169,23 +169,6 @@ Known issues
 1. The FSLeyes installation doesn't always work on Linux. Refer to the `FSLeyes installation guide <https://users.fmrib.ox.ac.uk/~paulmc/fsleyes/userdoc/latest/install.html>`_ if you need. In our testing, most issues came from the installation of the wxPython package.
 
 
-GPU-compatible installation
```

Comment:
That's done on the IVADOMED installation side right? Could we just link to the instructions on the IVADOMED page, in order to centralize the instructions since it may change in the future? https://ivadomed.org/installation.html#step-3-install-torch-and-torchvision-with-cpu-or-gpu-support

Reply:
Yes, I was just unsure on how to proceed because we have our own torch requirments in our environment file [here](https://github.com/neuropoly/axondeepseg/blob/57b658300974053615914410a767327c7a03e0a9/environment.yml#L31).
This is working on CPU only.

For GPU, I tested on rosenberg like this:
1) install ADS with the conda environment file according to our instructions
2) I pip install afterwards the GPU requirements from [here](https://ivadomed.org/installation.html#step-3-install-torch-and-torchvision-with-cpu-or-gpu-support), from the `Nvidia GPU Support` and `All OS` tabs. (currently `pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 --find-links https://download.pytorch.org/whl/torch_stable.html`)
It works well, the segmentation uses the GPU and all the tests passed.

I think my broader question is:
Should we have the torch requirements in our environment file even for CPU or always link to the ivadomed doc to always be up-to-date? We may want to check how they do it in SCT.

Reply:
Resolved